source/about/faq-product.rst

```diff
@@ -6,84 +6,175 @@ What features are available on different Mattermost clients?
++---------------------------------------------------------------------------------------------------+-------------+-------------+-----------------+
+| Message menus                                                                                     | |checkmark| | |checkmark| | |checkmark|     |
++---------------------------------------------------------------------------------------------------+-------------+-------------+-----------------+
+| Message actions                                                                                   | |checkmark| | |checkmark| |                 |
++---------------------------------------------------------------------------------------------------+-------------+-------------+-----------------+
+| Right-hand sidebar                                                                                | |checkmark| | |checkmark| |                 |
++---------------------------------------------------------------------------------------------------+-------------+-------------+-----------------+
+
+Authentication
+~~~~~~~~~~~~~~
++-------------------------------------------------------------------------------------+-------------+-------------+-------------+
+|                                       Feature                                       |     Web     |   Desktop   |    Mobile   |
++-------------------------------------------------------------------------------------+-------------+-------------+-------------+
+| :ref:`Email-password login <configure/authentication-configuration-settings:email>` | |checkmark| | |checkmark| | |checkmark| |
++-------------------------------------------------------------------------------------+-------------+-------------+-------------+
+| :ref:`AD/LDAP <onboard/ad-ldap:ad ldap setup>`                                      | |checkmark| | |checkmark| | |checkmark| |
++-------------------------------------------------------------------------------------+-------------+-------------+-------------+
+| :ref:`SAML SSO <onboard/sso-saml:saml single sign on>`                              | |checkmark| | |checkmark| | |checkmark| |
++-------------------------------------------------------------------------------------+-------------+-------------+-------------+
+| :ref:`GitLab SSO <onboard/sso-gitlab:gitlab single sign on>`                        | |checkmark| | |checkmark| | |checkmark| |
++-------------------------------------------------------------------------------------+-------------+-------------+-------------+
+| :ref:`Entra ID SSO <onboard/sso-entraid:entra id single sign on>`                   | |checkmark| | |checkmark| | |checkmark| |
++-------------------------------------------------------------------------------------+-------------+-------------+-------------+
+| :ref:`Google SSO <onboard/sso-google:google single sign on>`                        | |checkmark| | |checkmark| |             |
```

Comment:
@TomerPacific - There are 2 things happening to cause this not to work. 

1. A persnickety issue with Sphinx and relative links where the link text contains special characters (in this case ``AD/LDAP`` where ``/`` is the special character). 
2. The URL of that page is ``https://docs.mattermost.com/onboard/ad-ldap.html`` and not ``https://docs.mattermost.com/onboard/ad-ldap.html#ad-ldap-setup``. Sphinx knows the page URL based on its title, not by its section title. Any time you're referring to the page title and not a section, the `:doc:` syntax is used rather than `:ref:`.

For the first issue, one of ``/`` or ``-`` will work.